elyra/pipeline/processor_local.py

```diff
@@ -288,7 +300,7 @@ class PythonScriptOperationProcessor(ScriptOperationProcessor):
     _script_type = 'Python'
 
     def get_argv(self, file_path) -> List[str]:
-        return ['python3', file_path, '--PYTHONHOME', os.path.dirname(file_path)]
+        return ['python3', file_path]
```

Comment:
Per this comment, I think we should go with `sys.executable` here.
```suggestion
        return [f"{sys.executable}", file_path]
```